cacti/README.md

```diff
@@ -9,107 +9,142 @@ Get metrics from cacti service in real time to:
 
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
```

Comment:
Why is that removed? Should be moved instead?

Reply:
It's removed since in order to use the DD-cacti integration one needs to add `librrd` and `rrdtool`. Which is fine for a host install. But with a docker install it would mean build a custom Agent Image with those libs in it, we are going to create a custom FAQ for this, but in the meantime we remove this sentence that is misleading.